src/filters/dates/relative/relativeDates.test.ts

```diff
@@ -0,0 +1,216 @@
+import { DateTime, Settings } from 'luxon';
+import { TestCaseSchema } from '../../../serina.schema';
+import RelativeDates from './relativeDates';
+
+// Mock Date Time to Thu Jun 20 2019 08:34:52 GMT+0100
+Settings.now = () => 1561019692628;
+const currentYear = DateTime.utc().year;
+const currentMonth = DateTime.utc().month;
+const currentDay = DateTime.utc().day;
+
+beforeEach(() => {
+    // Restore mocked time in case we need to change it for individual tests
+    Settings.now = () => 1561019692628;
```

Comment:
> > > Maybe should match `2 weeks from today go to work` too?
> > 
> > 
> > Are you referring to the different order? That works. If you mean "from today", it is always the matter of simply adding more keywords to the list and then it will match.
> 
> It works for "2 weeks from now", but doesn't for "2 weeks from today"?

Because "from today" wasn't in the list of expected expressions, but I added it now. In the future, if you want to add one you can in relativeDates.constants.ts